terraform/modules/quorum-cluster-region/quorum-cluster-region-backup-event.tf

```diff
@@ -0,0 +1,329 @@
+       security_group_ids = ["${aws_security_group.allow_all.*.id}"]
+    }
+
+    environment {
+        variables {
+            NetworkId = "${var.network_id}"
+            Bucket = "${aws_s3_bucket.quorum_backup.id}"
+            Key = "${var.enc_ssh_key}"
+            SSHUser = "ubuntu"
+        }
+    }
+}
+
+resource "aws_iam_role" "backup_lambda" {
+  count = "${signum(lookup(var.maker_node_counts, var.aws_region, 0))}"
+  name = "iam_for_backup_lambda-${var.network_id}-${var.aws_region}"
+# See also https://aws.amazon.com/blogs/compute/easy-authorization-of-aws-lambda-functions/
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": "sts:AssumeRole",
+      "Principal": {
+        "Service": "lambda.amazonaws.com",
+        "Service": "events.amazonaws.com",
+        "Service": "sns.amazonaws.com"
+      },
+      "Effect": "Allow",
+      "Sid": ""
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_iam_policy" "backup_lambda_permissions" {
+  count = "${signum(lookup(var.maker_node_counts, var.aws_region, 0))}"
+  path = "/"
+  description = "IAM policy for accesing EC2 and S3 buckets from Lambda"
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": [
+        "ec2:*"
+      ],
+      "Effect": "Allow",
+      "Resource": "*"
+    },
+    {
+      "Action": ["s3:*"],
+      "Effect": "Allow",
+      "Resource": [
+        "${aws_s3_bucket.quorum_backup.arn}",
+        "${aws_s3_bucket.quorum_backup.arn}/*"
+      ]
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_iam_role_policy_attachment" "allow_backup_lambda_access_s3_and_ec2_resources" {
+   count      = "${signum(lookup(var.maker_node_counts, var.aws_region, 0))}"
+   role       = "${aws_iam_role.backup_lambda.name}"
+   policy_arn = "${aws_iam_policy.backup_lambda_permissions.arn}"
+}
+
+
+resource "aws_iam_policy" "allow_backup_lambda_logging" {
+  count = "${signum(lookup(var.maker_node_counts, var.aws_region, 0))}"
+  name = "BackupLambda-${var.network_id}-${var.aws_region}"
+  path = "/"
+  description = "IAM policy for logging from a lambda"
+
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": [
+        "logs:*"
+      ],
+      "Resource": "arn:aws:logs:*:*:*",
+      "Effect": "Allow"
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_iam_role_policy_attachment" "allow_backup_lambda_logging" {
+   count = "${signum(lookup(var.maker_node_counts, var.aws_region, 0))}"
+   role = "${aws_iam_role.backup_lambda.name}"
+   policy_arn = "${aws_iam_policy.allow_backup_lambda_logging.arn}"
+}
+
+resource "null_resource" "fetch_backup_lambda_zip" {
+  triggers { always="${uuid()}"}
+  provisioner "local-exec" {
+     command = <<EOT
+ if [ ! -e ${var.aws_region}-${var.backup_lambda_output_path} ]; then 
+   wget -O ${var.aws_region}-${var.backup_lambda_output_path} https://github.com/EximChua/BackupLambda/releases/download/0.1/BackupLambda.zip
```

Comment:
If you have a URL parameter shouldn't you be using it here?

Reply:
Fixed